### DISCUSSION TITLE
Welcome to CKAN Discussions!

### DISCUSSION THREAD

**Discussion by HebaruSan** [KSP-CKAN/CKAN#3232] [General]
## Hi all

This is a new GitHub feature that looks like it might be useful for CKAN users and developers. We're going to try it out and see how it goes.

Below is the default intro text that GitHub fills in for us, left unchanged so you can see the idea behind it. Don't feel obligated to introduce yourself. :wink: 

____

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by techman83** [comment]
Oh interesting. Forums for GitHub, neat. :wave:

**Comment by crazydied2** [comment]
So, might just be me, but I can't seem to understand what I am doing when I need to add a KSP instance, what file am I supposed to be selecting?

**Reply by HebaruSan** [reply]
Assuming you're in the GUI, you're supposed to pick the buildID.txt or buildID64.txt file.

**Reply by crazydied2** [reply]
Alright, thank you.

**Reply by SeattleRex** [reply]
It wasn't just you, I just had that issue as well.  I wasn't sure what I was supposed to give it as a base. I noticed that it was screening for text files, though, so I picked the only one that was in the base application directory. All's well that ends well.

**Reply by dimasick12345** [reply]
Help! I ran SCAN on version ksp 1.3.1 and it gives me the following error: Failed to download https://github.com/KSP-CKAN/CKAN-meta/archive/master.tar.gz:

**Comment by APAeroDan** [comment]
I just installed KSP on linux for the first time, and I installed ckan from the repository, however, I can't seem to figure out how to start it...  I also can't figure out what directory its installed in.
Seems much more straightforward in windows, but KSP runs better in linux.

**Reply by DasSkelett** [reply]
A launch script gets installed to `/usr/bin/ckan` , `ckan.exe` gets installed to `/usr/lib/ckan/ckan.exe`.
It also installs  `.desktop` files to `/usr/share/applications/ckan.desktop` and `/usr/share/applications/ckan-consoleui.desktop`, so you should find two CKAN entries in your launcher / application starter / system menu / start menu, one for the GUI and one for the ConsoleUI.

**Reply by HebaruSan** [reply]
:point_up:  That's assuming you installed the .deb or .rpm files, btw. If you just downloaded the .exe, then it's wherever your browser put it, probably your Downloads folder.

**Comment by APAeroDan** [comment]
Ok, thanks for the tip. I'll check it out.

**Comment by nar664** [comment]
i cant find which link to click to download

**Reply by Postremus** [reply]
On the main github page of ksp-ckan (https://github.com/KSP-CKAN/CKAN), click on the "Releases" link. Its on the right side if you are on desktop. There you can find the exe for the latest release if you scroll down a bit.

**Reply by HebaruSan** [reply]
Or click the big Download button on that same page.
<img width="306" height="69" alt="image" src="https://github.com/user-attachments/assets/a7a1e687-48fe-4a42-a12c-76df9f2e0052" />

**Comment by nar664** [comment]
where is the .exe

**Comment by keshavarziana505-sketch** [comment]
Every time I run ckan it says you must run it without administrator user
But I used to download mods with the same user 
And I can't understand why it's doing this
